src/blog/2024/01/history-of-nodered.md

```diff
@@ -0,0 +1,46 @@
+---
+title: History of Node-RED
+subtitle: How it all started as told by Node-RED creator Nick O'Leary
+description: How it all started by Node-RED creator Nick O'Leary
+date: 
+authors: 
+image: 
```

Comment:
@Hasmin-AC Can you create a request for a social tile through the art request process? https://flowfuse.com/handbook/design/art-requests/#art-requests